spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -23,20 +23,26 @@
   scenario "see a payment breakdown per NPQ course and a payment breakdown of each individual NPQ courses for each provider" do
     given_i_am_logged_in_as_a_finance_user
     and_there_is_npq_provider_with_contracts
-    and_those_courses_have_submitted_declations
+    and_those_courses_have_submitted_declarations
     when_i_visit_the_payment_breakdown_page
     and_choose_to_see_npq_payment_breakdown
     and_i_select_an_npq_lead_provider
     then_i_should_have_the_correct_payment_breakdown_per_npq_lead_provider
+    and_the_page_should_be_accessible
```

Comment:
if you're doing percy or axe i think you need `js: true` otherwise it'll run through rack test which isn't a browser

Reply:
Of course, yes! thanks!